view/utils/init-table-sorting-on-client.js

```diff
@@ -16,6 +16,23 @@ module.exports = function (domElement/*, opts */) {
 		// set type, either numeric or text 
 		type: 'numeric'
 	});
+	window.jQuery.tablesorter.addParser({
+		// set a unique id 
+		id: 'dates',
+		is: function (s) {
+			// return false so this parser is not auto detected 
+			return false;
+		},
+		format: function (item) {
+			var parsedItem = item;
+			if (item.indexOf('/') !== -1) {
+				parsedItem = item.replace(/\/+/g, '-').split('-').reverse().join('-');
```

Comment:
@mtuchowski brief search did not bring anything interesting. Still it didn't work ouside of the box for dbjs Date... even though it stringified itself to that...